py/server/deephaven/table.py

```diff
@@ -166,9 +167,18 @@ def where(self, filters: Union[str, Filter, Sequence[str], Sequence[Filter]]):
         j_filter_ops_recorder = jpy.cast(self.j_node_ops_recorder, _JFilterOperationsRecorder)
         return self.__class__(j_filter_ops_recorder.where(and_(filters).j_filter))
 
+class _UpdateViewOperationsRecorder(Protocol):
+    """A mixin for creating updateView operations to be applied to individual nodes of RollupTable."""
+
+    def update_view(self, formulas: Union[str, Sequence[str]]):
+        """Returns a new recorder with the :meth:`~deephaven.table.Table.update_view` operation applied to nodes."""
+        formulas = to_sequence(formulas)
+        j_update_view_ops_recorder = jpy.cast(self.j_node_ops_recorder, _JUpdateViewOperationsRecorder)
+        return self.__class__(j_update_view_ops_recorder.updateView(*formulas))
+
 
 class RollupNodeOperationsRecorder(JObjectWrapper, _FormatOperationsRecorder,
-                                   _SortOperationsRecorder):
+                                   _SortOperationsRecorder, _UpdateViewOperationsRecorder):
```

Comment:
Pointing out that this is a public class with private parents.
@jmao-denver should assess.

Reply:
This is by design. The class level docstring  includes this note;
```    Note: It should not be instantiated directly. User code must call :meth:`~RollupTable.node_operation_recorder` to
    create an instance of the recorder.```